torch/csrc/jit/runtime/profiling_graph_executor_impl.cpp

```diff
@@ -382,12 +399,16 @@ ExecutionPlan ProfilingGraphExecutorImpl::getPlanFor(
   std::lock_guard<std::mutex> lock(compile_mutex);
   GRAPH_DEBUG("Running ProfilingGraphExecutorImpl ", this);
 
+  if (!remaining_bailout_depth_.has_value() || !tensorExprFuserEnabled()) {
+    remaining_bailout_depth_ = remaining_bailout_depth;
```

Comment:
Could you add a comment here ? Why is this here ?

Reply:
yeah let me add a comment